samples/android/app/src/main/res/layout/activity_anoncreds.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:paddingTop="4dp"
+    android:paddingBottom="16dp">
+
+    <ProgressBar
+        android:id="@+id/pbAnoncreds"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_marginTop="-7dp"
+        android:backgroundTint="@color/disabled"
+        android:indeterminateOnly="true"
+        android:indeterminateTint="@color/buttonPrimary"
+        android:visibility="invisible"
+        tools:visibility="visible"/>
+
+
+    <ScrollView
```

Comment:
why does this not attach to progress bar? probably the same reason that you use negative margin above?

you should attach to progress bar and change paddings to it instead.

if that does not work you should use framelayout instead constraint layout